requirements.txt

```diff
@@ -0,0 +1 @@
+th2-data-services-lwdp==3.1.0.1
```

Comment:
I think we shouldn't have the relation to th2-data-services-lwdp.

1. The wrong version - We actually used https://github.com/th2-net/th2-ds-source-lwdp/blob/release-2.1 for RDP-5
2. We cannot have 2 separate independent data-source plugins, due to this relation.

Solution:
Just copy lwdp-2.1 code to this repo